assets/css/style.css

```diff
@@ -1,276 +1,428 @@
-.header-container {
+body {
+  background-color: var(--white);
+  width: 100%;
+  font-family: var(--font-family-style);
+  position: relative;
+}
+
+.top-mob {
+  height: 718px;
+  background-image: url('/assets/background-desk.png');
+  background-size: cover;
```

Comment:
- [ ] [optional] You can use the ``` background-size: contain ``` to replicat the background from the Figma design

 - Figma Design:
 ![image](https://user-images.githubusercontent.com/68796232/167130347-01aaeaa9-2330-4d4b-b336-3b517177368a.png)

- Your Design:
![image](https://user-images.githubusercontent.com/68796232/167130321-93f908e3-f83d-4584-a8ce-a8338882f43b.png)